src/guides/v2.3/ui_comp_guide/concepts/knockout-bindings.md

```diff
@@ -346,7 +346,7 @@ Callback that is invoked when user clicks outside of the element.
 
 ```html
 <div id="target" outerClick="function () {
-    console.log('Clicked outside of the "target" node.');
```

Comment:
Hi @niravkrish. I believe that the "target" with quotes is a reference to the DOM element with `id="target"`. So, by removing the quotes we might reduce the clarity of the statement inside `console.log` invocation. I propose either leave the quotes or add an additional clarification in the following manner

```javascript 
    console.log('Clicked outside of the node with id="target"');
```

Thank you.

Reply:
Hi @niravkrish. Thank you for the change. I'm not sure it will work in case of `console.log()`

![image](https://user-images.githubusercontent.com/2148959/72594697-5a8e0580-3908-11ea-89c1-a991c0506844.png)

Reply:
- I have tested it and we can definitely get the exact result.
- you have tested it in console but it will work in html.
- you can test it in html on-click element. 
- Please see below screenshot and example

<a href='https://codepen.io/mage2nirav/pen/gObQYMy'>See the example code in codepen</a>

![screenshot-codepen io-2020 01 19-17_47_51](https://user-images.githubusercontent.com/44108490/72680894-e0ff3e80-3ae4-11ea-8b5e-95ea85740931.png)

Thank You
Nirav Patel

Reply:
You are right. If we put this message into an HTML node it will be shown in the console as expected. Thank you for the clarification.